Mathlib/Algebra/Polynomial/RingDivision.lean

```diff
@@ -130,6 +130,17 @@ theorem mem_nonZeroDivisors_of_leadingCoeff {p : R[X]} (h : p.leadingCoeff ∈ R
 
 end nonZeroDivisors
 
+section IsSimpleRing
+
+variable [IsSimpleRing R] [CommRing S] [Nontrivial S] [Algebra R S]
+
+theorem natDegree_pos_of_monic_of_root {p : R[X]} (hp : p.Monic) {x : S} (hx : aeval x p = 0) :
```

Comment:
Same here: instead of assuming [IsSimpleRing R] [Nontrivial S], assuming [FaithfulSMul R S] would be more general.